pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxReceiveOperator.java

```diff
@@ -164,7 +164,6 @@ protected TransferableBlock getNextBlock() {
               return block;
             } else {
               if (!block.getResultMetadata().isEmpty()) {
-                _operatorStats.clearExecutionStats();
```

Comment:
why is this necessary previously? i thought the stats aggregator is only created once right?

Reply:
Previously we were aggregating stats, hence there was need to clear them. Now, we are just replacing the stats.